themes/bootstrap3/templates/RecordDriver/DefaultRecord/link-citedBy.phtml

```diff
@@ -0,0 +1,15 @@
+<?php
+echo htmlspecialchars(
+    $this->url(
+        $this->searchOptions($this->driver->getSearchBackendIdentifier())->getCitedByAction(),
+        [],
+        [
+            'query' => [
+                'lookfor' => '"' . $this->driver->getUniqueID() . '"',
```

Comment:
If we're going to put this in the DefaultRecord space, would it be better to have a method in Options or Params for building the query, rather than hard-coding implementation details in the template? (And obviously, if we change this here, we should do the same below).

I don't insist on changing this, but it seems a little weird to have hard-coded field names here, since those details would almost certainly be different on a different platform.

Reply:
I can't see a precedent for this kind of method in Options, do we have any? 

I considered these sort of virtual fields that could be mapped to the real fields just like the search types in other link templates. If we had this functionality for Solr as well, we could move these files to Primo and replace these with the Solr ones. But I'd like to keep the templates here to reserve the link types.

Reply:
That's fair. I was thinking that this was just performing a standard search action because of its use of standard lookfor/filter query parameters, but since it has its own citedBy action, it would be possible to do remapping there as necessary. No objection to leaving as-is.